src/creatures/combat/combat.cpp

```diff
@@ -961,9 +961,9 @@ void Combat::setupChain(const std::shared_ptr<Weapon> &weapon) {
 		0, 0, 0, 1, 0, 0, 0,
 	};
 	// clang-format on
-	auto area = std::make_unique<AreaCombat>();
-	area->setupArea(areaList, 7);
-	setArea(area);
+	auto m_area = std::make_unique<AreaCombat>();
```

Comment:
The name "m_" is usually used in class member variables, which is not the case here. I believe it can cause confusion.